paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/PaimonScan.scala

```diff
@@ -39,7 +40,7 @@ case class PaimonScan(
     override val pushDownLimit: Option[Int],
     bucketedScanDisabled: Boolean = false)
```

Comment:
`disableBucketedScan` is used in spark3.2, and `bucketedScanDisabled` in other `SparkScan`. Refine this,plz.